lib/stop_watch_timer.dart

```diff
@@ -109,8 +111,8 @@ class StopWatchTimer {
   int _startTime = 0;
   int _stopTime = 0;
   late int _presetTime;
-  int? _second;
-  int? _minute;
+  int _second = 0;
+  int _minute = 0;
   List<StopWatchRecord> _records = [];
```

Comment:
_second and _minute changed from int? to int to avoid double emission of the initial value. Otherwise, if they start with null, the initial value will be emitted when BehaviorSubject.seeded is called and when it is first compared against latestSecond or latestMinute.